test/statsCases/filter-warnings/webpack.config.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+const webpack = require("webpack");
+
+const baseConfig = {
+	entry: "./index",
+	output: {
+		filename: "bundle.js"
+	},
+	plugins: [new webpack.optimize.UglifyJsPlugin({
+		sourceMap: true,
+		compress: {
+			warnings: true,
+		},
+		mangle: false,
+		beautify: true,
+		comments: false
+	})],
+	stats: {
+		chunkModules: false,
```

Comment:
`modules: false` `hash: false` `assets: false` to make the stats smaller

This will cause less changes when unrelated code is updated